sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/TransformWithStateExec.scala

```diff
@@ -44,23 +48,25 @@ import org.apache.spark.util.CompletionIterator
  * @param batchTimestampMs processing timestamp of the current batch.
  * @param eventTimeWatermarkForLateEvents event time watermark for filtering late events
  * @param eventTimeWatermarkForEviction event time watermark for state eviction
+ * @param isStreaming defines whether the query is streaming or batch
  * @param child the physical plan for the underlying data
  */
 case class TransformWithStateExec(
-    keyDeserializer: Expression,
-    valueDeserializer: Expression,
-    groupingAttributes: Seq[Attribute],
-    dataAttributes: Seq[Attribute],
-    statefulProcessor: StatefulProcessor[Any, Any, Any],
-    timeoutMode: TimeoutMode,
-    outputMode: OutputMode,
-    keyEncoder: ExpressionEncoder[Any],
-    outputObjAttr: Attribute,
-    stateInfo: Option[StatefulOperatorStateInfo],
-    batchTimestampMs: Option[Long],
-    eventTimeWatermarkForLateEvents: Option[Long],
-    eventTimeWatermarkForEviction: Option[Long],
-    child: SparkPlan)
+     keyDeserializer: Expression,
```

Comment:
Is the indent change intentional ?

Reply:
Was trying to follow this:

> For method declarations, use 4 space indentation for their parameters and put each in each line when the parameters don't fit in two lines. Return types can be either on the same line as the last parameter, or start a new line with 2 space indent.

https://github.com/databricks/scala-style-guide?tab=readme-ov-file#indent

Reply:
Didn't realize I had deviated from this, thanks for catching it.